extensions/typescript-language-features/src/languageFeatures/updatePathsOnRename.ts

```diff
@@ -56,38 +56,39 @@ class UpdateImportsOnFileRenameHandler extends Disposable {
 		super();
 
 		this._register(vscode.workspace.onDidRenameFiles(async (e) => {
-			const [{ newUri, oldUri }] = e.files;
-			const newFilePath = this.client.toTsFilePath(newUri);
-			if (!newFilePath) {
-				return;
-			}
+			for (const { newUri, oldUri } of e.files) {
+				const newFilePath = this.client.toTsFilePath(newUri);
+				if (!newFilePath) {
+					return;
```

Comment:
These should likely be `continue` instead `return` to avoid bailing if just one file is bad

Reply:
Done